python/ray/serve/application.py

```diff
@@ -0,0 +1,376 @@
+from typing import List, Dict, TextIO, Optional, Union
+import json
+import yaml
+import time
+import sys
+
+import ray
+from ray import serve
+from ray.serve.api import (
+    Deployment,
+    _get_global_client,
+)
+from ray.serve.common import DeploymentStatus
+from ray.dashboard.modules.serve.schema import (
+    ServeApplicationSchema,
+    serve_application_to_schema,
+    schema_to_serve_application,
+    serve_application_status_to_schema,
+)
+from ray.dashboard.modules.serve.sdk import ServeSubmissionClient
+from ray.serve.utils import logger
+
+
+class Application:
+    """
+    Contains a group of Serve deployments and manages their deployment to the
+    Ray cluster.
```

Comment:
best practice is for the first line of the docstring to be a single sentence that fits on one line, like:

```
"""Contains a group of Serve deployments and can be used to deploy, update, and monitor them.

More details here. Blah, blah, blah.
"""
```

Reply:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

please try to conform to this with all of the docstrings written here